src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
```

Comment:
100 should be made a constant field (private static final field with name in UPPER_CASE)
https://www.javatpoint.com/java-constant